index.src.html

```diff
@@ -1380,6 +1380,28 @@ <h4 id="body-extraction" algorithm>
           6.  Set |Content-Type| to
               `application/x-www-form-urlencoded;charset=UTF-8`.
 
+          Otherwise, |data| is a generic object, so:
+
+          1.  If |data| has a property named like the |credential|'s
+              {{PasswordCredential/idName}} or
+              {{PasswordCredential/passwordName}}, <a>throw</a> a `TypeError`
+              exception.
```

Comment:
Actually, @tobie and/or @domenic might know a better way to copy/modify an object without triggering unwanted side-effects. I'd wouldn't be surprised if there was an existing concept in WebIDL or ECMAScript that I'm unaware of.